Fixed typo in py9b.link.ble among other things

HelaFaye opened the pull request:
Even if you only take the changes applied to ble, I think it would help people who wish to make use of py9b.

HelaFaye committed:
Added custom UUID parameters for fleet vehicles

HelaFaye committed:
Updated and proofread

HelaFaye committed:
Changed all IPs to localhost and added contribs

HelaFaye committed:
fixed Queue usage in BLE

flowswitch commented:
- "fleet vehicles" - stealing is NOT WELCOME here.

- What's the point of changing all IPs to localhost? Some people are using the TCP link over WiFi, others are using it over ADB forward - who is "right" ? :)

- "queue" with lower case "q" is correct module name in older Python versions (i.e. 2.7.14). The correct way to support both of them would be something like this:
`
try: 
	import queue
except ImportError:
	import Queue as queue
`

HelaFaye commented:
The fleet vehicles I've reprogrammed using blefleet have not been theft and we're decommissioned for stupid reasons like battery undervoltage. Why would someone use an older version of Python? If that is the right way to support both, then I don't understand why it's not already written like that.

HelaFaye committed:
import Queue exception handling

joeybab3 commented:
Some vehicles from "fleets" can be acquired 100% legally either by company decommissioning or by police impound auctions.

HelaFaye committed:
added changesn back

Informatic committed:
fix shit

HelaFaye committed:
begin python 2to3 migration (automated)

Informatic committed:
cleanups for py3 migration

Informatic committed:
py9b/link/bleak: asyncio & bleak-based link implementation

Informatic committed:
py9b/transport/xiaomi: BLE encryption support

HelaFaye committed:
Merged contributions from Informatic/py9b

Informatic committed:
py9b/transport/ninebot: fix python3 supprt

Informatic committed:
code style cleanup

HelaFaye committed:
been working on droidble in NineRiFt

HelaFaye committed:
merge portions of Informatic's contributions that have been reviewed

HelaFaye committed:
remove extra line

Informatic committed:
py9b/transport/xiaomi: add recover_keys method

Informatic committed:
py9b/link/bleak: proper advertisement matching

Informatic committed:
cli: WIP click-based CLI tool

Informatic committed:
py9b/command/regio: fix write command on ninebot > 0301

Somewhere between ESC 0301 and 0401 response format for write command
has changed and is much more similar to that of xiaomi - arg contains
register address and single response data byte is a write result. This
is behaviour that matches description in nbdocs.pdf on scooterhacking
wiki.

Informatic committed:
cli: add a bunch of extra commands

Informatic committed:
cli: cleanups

Informatic committed:
fwupd: fix after py3k migration

Sadly bleak link implementation doesn't seem to work good enough to
flash yet, but fwupd of ESC and BLE over serial has been confirmed to
work fine.

Informatic committed:
py9b/link/bleak: use writes with confirmations

This fixes some bluez compatiblity issues and improves connection
stability.

Informatic committed:
py9b/transport/base: add optional recv/parse retries

Informatic committed:
cli: add changesn command

Informatic committed:
Merge branch 'master' of https://github.com/slinger360/py9b

HelaFaye committed:
Merge pull request #1 from Informatic/master

A bunch of cleanups

HelaFaye committed:
added return to read and some feedback

HelaFaye committed:
changed BleakLink to BLELink

HelaFaye committed:
moved all usage examples to py9b/examples

HelaFaye committed:
restructured

HelaFaye committed:
restructured

HelaFaye committed:
once more

HelaFaye committed:
separated scooter identity bytearrays

HelaFaye committed:
changed connection confirmations to use self.connected.is_set()

Informatic committed:
ble: fix writes after reformatting

Informatic committed:
bleak: fix big writes

HelaFaye committed:
fixed bleak implementation with Informatic's help

HelaFaye committed:
Merge pull request #2 from Informatic/fix-big-writes

Fix big writes

HelaFaye committed:
moved cli and fwupd
